package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "todo-list-project",
+  "version": "1.0.0",
+  "description": "this is a simple todo list application using webpack and it is being served by webpack dev server",
+  "private": true,
+  "scripts": {
```

Comment:
- [ ] A local server cannot be started within your application. Please add start options to your scripts to enable your application to be run using the command `npm start`, `npm run start` or `npm serve`.

**Package config error:**
![image](https://user-images.githubusercontent.com/12745474/134149164-5197482d-4a66-4d7d-ab06-d5c1a616529d.png)

Reply:
Thanks for picking this out. It has been resolved